[BUG] Edit Profile Error 500

### Description

The Edit Profile screen shows up as blank in both local and reverse proxy. When user goes directly to /api/profile/ Error 500 is displayed

![Image](https://github.com/user-attachments/assets/37a85a7e-ff1b-498b-a401-22a3c3823b8e)
![Image](https://github.com/user-attachments/assets/481c5250-5c07-4564-b890-e7506d7e5c41)

![Image](https://github.com/user-attachments/assets/57a7056c-96f8-46ba-8997-a03f72b24930)

### Steps to reproduce

1. Login to Paperless
2. Click on user
3. Click on My Profile

### Webserver logs

```bash
2025-05-04T14:22:25.987277884Z [2025-05-04 14:22:25,985] [ERROR] [django.request] Internal Server Error: /api/profile/
2025-05-04T14:22:25.987303949Z Traceback (most recent call last):
2025-05-04T14:22:25.987307982Z   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
2025-05-04T14:22:25.987311328Z     raise exc_info[1]
2025-05-04T14:22:25.987314358Z   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 42, in inner
2025-05-04T14:22:25.987317549Z     response = await get_response(request)
2025-05-04T14:22:25.987320506Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:22:25.987323432Z   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
2025-05-04T14:22:25.987326631Z     raise exc_info[1]
2025-05-04T14:22:25.987329591Z   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
```

### Browser logs

```bash
api/profile/:1 - Failed to load resource: the server responded with a status of 500 ()
main.js:3  ERROR I2
handleError @ main.js:3
vcd15cbe7772f49c399c6a5babf22c1241717689176015:1 - Failed to load resource: net::ERR_NAME_NOT_RESOLVED
```

### Paperless-ngx version

2.15.3

### Host OS

Ubuntu 25.04

### Installation method

Docker - official image

### System status

```json
{
    "pngx_version": "2.15.3",
    "server_os": "Linux-6.14.0-15-generic-x86_64-with-glibc2.36",
    "install_type": "docker",
    "storage": {
        "total": 105089261568,
        "available": 42822356992
    },
    "database": {
        "type": "mysql",
        "url": "paperless",
        "status": "OK",
        "error": null,
        "migration_status": {
            "latest_migration": "paperless_mail.0029_mailrule_pdf_layout",
            "unapplied_migrations": []
        }
    },
    "tasks": {
        "redis_url": "redis:///1:6379",
        "redis_status": "OK",
        "redis_error": null,
        "celery_status": "OK",
        "celery_url": "celery@4cd7551ede5d",
        "celery_error": null,
        "index_status": "OK",
        "index_last_modified": "2025-05-04T13:29:40.048111Z",
        "index_error": null,
        "classifier_status": "OK",
        "classifier_last_trained": "2025-05-04T14:05:16.909695Z",
        "classifier_error": null,
        "sanity_check_status": "OK",
        "sanity_check_last_run": "2025-05-04T00:30:28.627713Z",
        "sanity_check_error": null
    }
}
```

### Browser

Edge and Chrome

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

Please see full logs below:

2025-05-04T14:23:40.340154804Z [init-start] paperless-ngx docker container starting...
2025-05-04T14:23:40.343279010Z [init-start]  paperless-ngx docker container starting init as root
2025-05-04T14:23:40.347681691Z [env-init] Checking for environment from files
2025-05-04T14:23:40.354036717Z [env-init] PAPERLESS_ADMIN_PASSWORD set from PAPERLESS_ADMIN_PASSWORD_FILE
2025-05-04T14:23:40.358504175Z [env-init] PAPERLESS_ADMIN_USER set from PAPERLESS_ADMIN_USER_FILE
2025-05-04T14:23:40.363080198Z [env-init] PAPERLESS_DBPASS set from PAPERLESS_DBPASS_FILE
2025-05-04T14:23:40.370563332Z [env-init] PAPERLESS_DBUSER set from PAPERLESS_DBUSER_FILE
2025-05-04T14:23:40.375199315Z [env-init] PAPERLESS_SECRET_KEY set from PAPERLESS_SECRET_KEY_FILE
2025-05-04T14:23:40.381741457Z [init-tesseract-langs] Checking if additional teseract languages needed
2025-05-04T14:23:40.381754900Z [init-tesseract-langs] No additional installs requested
2025-05-04T14:23:40.385809713Z [init-db-wait] Waiting for MariaDB to report ready
2025-05-04T14:23:40.385823325Z [init-db-wait] Waiting for MariaDB to start...
2025-05-04T14:23:40.386437620Z Connected to MariaDB
2025-05-04T14:23:40.386445194Z [init-db-wait] Database is ready
2025-05-04T14:23:40.390339679Z [init-user] No UID changes for paperless
2025-05-04T14:23:40.390359179Z [init-user] No GID changes for paperless
2025-05-04T14:23:40.391297088Z [init-redis-wait] Waiting for Redis to report ready
2025-05-04T14:23:40.395026530Z [init-folders] Running with root privileges, adjusting directories and permissions
2025-05-04T14:23:40.399863067Z mkdir: created directory '/tmp/paperless'
2025-05-04T14:23:40.471753665Z changed ownership of '/tmp/paperless' from root:root to paperless:paperless
2025-05-04T14:23:40.476765351Z [init-migrations] Apply database migrations...
2025-05-04T14:23:41.127220827Z Waiting for Redis...
2025-05-04T14:23:41.134336927Z Connected to Redis broker.
2025-05-04T14:23:41.164225580Z [init-redis-wait] Redis ready
2025-05-04T14:23:44.219956221Z Operations to perform:
2025-05-04T14:23:44.219978414Z   Apply all migrations: account, admin, auditlog, auth, authtoken, contenttypes, django_celery_results, documents, guardian, mfa, paperless, paperless_mail, sessions, socialaccount
2025-05-04T14:23:44.290249959Z Running migrations:
2025-05-04T14:23:44.290270920Z   No migrations to apply.
2025-05-04T14:23:44.664111363Z [init-superuser] Creating superuser...
2025-05-04T14:23:50.036965387Z Did not create superuser, a user alexander already exists
2025-05-04T14:23:50.321546461Z [init-superuser] Superuser creation done
2025-05-04T14:23:50.326984294Z [init-checks] Running Django checks
2025-05-04T14:23:55.579919074Z System check identified no issues (0 silenced).
2025-05-04T14:23:55.872968556Z [custom-init] /custom-cont-init.d doesn't exist, nothing to do
2025-05-04T14:23:55.879672351Z [init-complete] paperless-ngx docker container init completed in 15 seconds
2025-05-04T14:23:55.879690986Z [init-complete] Starting services
2025-05-04T14:23:55.902280039Z [svc-flower] Checking if we should start flower...
2025-05-04T14:23:55.902301099Z [svc-flower] Starting flower in 5s
2025-05-04T14:23:56.900144765Z [INFO] Starting granian (main PID: 232)
2025-05-04T14:23:56.900393037Z [INFO] Listening at: http://:::8000
2025-05-04T14:23:56.905550161Z [INFO] Spawning worker-1 with PID: 255
2025-05-04T14:23:56.911179819Z [INFO] Spawning worker-2 with PID: 257
2025-05-04T14:24:00.637421300Z celery beat v5.5.1 (immunity) is starting.
2025-05-04T14:24:07.948279156Z [2025-05-04 14:24:07,947] [INFO] [paperless.management.consumer] Polling directory for changes: /usr/src/paperless/consume
2025-05-04T14:24:08.442901566Z __    -    ... __   -        _
2025-05-04T14:24:08.442925038Z LocalTime -> 2025-05-04 14:24:08
2025-05-04T14:24:08.442929891Z Configuration ->
2025-05-04T14:24:08.442933701Z     . broker -> redis://redis:6379/1
2025-05-04T14:24:08.442937342Z     . loader -> celery.loaders.app.AppLoader
2025-05-04T14:24:08.442941286Z     . scheduler -> celery.beat.PersistentScheduler
2025-05-04T14:24:08.442944959Z     . db -> /usr/src/paperless/data/celerybeat-schedule.db
2025-05-04T14:24:08.442948654Z     . logfile -> [stderr]@%INFO
2025-05-04T14:24:08.443031910Z     . maxinterval -> 5.00 minutes (300s)
2025-05-04T14:24:08.444114141Z [2025-05-04 14:24:08,443] [INFO] [celery.beat] beat: Starting...
2025-05-04T14:24:09.215551026Z  
2025-05-04T14:24:09.215573016Z  -------------- celery@d6cc463bca38 v5.5.1 (immunity)
2025-05-04T14:24:09.215577725Z --- ***** ----- 
2025-05-04T14:24:09.215581559Z -- ******* ---- Linux-6.14.0-15-generic-x86_64-with-glibc2.36 2025-05-04 14:24:09
2025-05-04T14:24:09.215585244Z - *** --- * --- 
2025-05-04T14:24:09.215603969Z - ** ---------- [config]
2025-05-04T14:24:09.215608410Z - ** ---------- .> app:         paperless:0x785cd84762a0
2025-05-04T14:24:09.215612482Z - ** ---------- .> transport:   redis://redis:6379/1
2025-05-04T14:24:09.215616173Z - ** ---------- .> results:     
2025-05-04T14:24:09.215619764Z - *** --- * --- .> concurrency: 2 (prefork)
2025-05-04T14:24:09.215623506Z -- ******* ---- .> task events: ON
2025-05-04T14:24:09.215627267Z --- ***** ----- 
2025-05-04T14:24:09.215630864Z  -------------- [queues]
2025-05-04T14:24:09.215634509Z                 .> celery           exchange=celery(direct) key=celery
2025-05-04T14:24:09.215638242Z                 
2025-05-04T14:24:09.215641816Z 
2025-05-04T14:24:09.215666323Z [tasks]
2025-05-04T14:24:09.215671410Z   . documents.bulk_edit.delete
2025-05-04T14:24:09.215674411Z   . documents.signals.handlers.send_webhook
2025-05-04T14:24:09.215677258Z   . documents.tasks.bulk_update_documents
2025-05-04T14:24:09.215680057Z   . documents.tasks.check_scheduled_workflows
2025-05-04T14:24:09.215682927Z   . documents.tasks.consume_file
2025-05-04T14:24:09.215685744Z   . documents.tasks.empty_trash
2025-05-04T14:24:09.215688532Z   . documents.tasks.index_optimize
2025-05-04T14:24:09.215691328Z   . documents.tasks.sanity_check
2025-05-04T14:24:09.215694094Z   . documents.tasks.train_classifier
2025-05-04T14:24:09.215696918Z   . documents.tasks.update_document_content_maybe_archive_file
2025-05-04T14:24:09.215699755Z   . paperless_mail.mail.apply_mail_action
2025-05-04T14:24:09.215702544Z   . paperless_mail.mail.error_callback
2025-05-04T14:24:09.215705372Z   . paperless_mail.tasks.process_mail_accounts
2025-05-04T14:24:09.215708229Z 
2025-05-04T14:24:09.418047871Z [2025-05-04 14:24:09,417] [INFO] [celery.worker.consumer.connection] Connected to redis://redis:6379/1
2025-05-04T14:24:09.437885699Z [2025-05-04 14:24:09,437] [INFO] [celery.apps.worker] celery@d6cc463bca38 ready.
2025-05-04T14:24:10.664706036Z [2025-05-04 14:24:10,664] [INFO] [pikepdf._core] pikepdf C++ to Python logger bridge initialized
2025-05-04T14:24:11.407362066Z [2025-05-04 14:24:11,407] [INFO] [flower.command] Visit me at http://0.0.0.0:5555
2025-05-04T14:24:11.443520973Z [2025-05-04 14:24:11,443] [INFO] [flower.command] Broker: redis://redis:6379/1
2025-05-04T14:24:11.457457265Z [2025-05-04 14:24:11,457] [INFO] [flower.command] Registered tasks: 
2025-05-04T14:24:11.457472830Z ['celery.accumulate',
2025-05-04T14:24:11.457476503Z  'celery.backend_cleanup',
2025-05-04T14:24:11.457479447Z  'celery.chain',
2025-05-04T14:24:11.457482560Z  'celery.chord',
2025-05-04T14:24:11.457485625Z  'celery.chord_unlock',
2025-05-04T14:24:11.457488460Z  'celery.chunks',
2025-05-04T14:24:11.457491262Z  'celery.group',
2025-05-04T14:24:11.457494067Z  'celery.map',
2025-05-04T14:24:11.457496871Z  'celery.starmap',
2025-05-04T14:24:11.457499674Z  'documents.bulk_edit.delete',
2025-05-04T14:24:11.457502543Z  'documents.signals.handlers.send_webhook',
2025-05-04T14:24:11.457505483Z  'documents.tasks.bulk_update_documents',
2025-05-04T14:24:11.457508314Z  'documents.tasks.check_scheduled_workflows',
2025-05-04T14:24:11.457511174Z  'documents.tasks.consume_file',
2025-05-04T14:24:11.457513985Z  'documents.tasks.empty_trash',
2025-05-04T14:24:11.457516768Z  'documents.tasks.index_optimize',
2025-05-04T14:24:11.457519592Z  'documents.tasks.sanity_check',
2025-05-04T14:24:11.457538229Z  'documents.tasks.train_classifier',
2025-05-04T14:24:11.457541378Z  'documents.tasks.update_document_content_maybe_archive_file',
2025-05-04T14:24:11.457544333Z  'paperless_mail.mail.apply_mail_action',
2025-05-04T14:24:11.457547135Z  'paperless_mail.mail.error_callback',
2025-05-04T14:24:11.457549970Z  'paperless_mail.tasks.process_mail_accounts']
2025-05-04T14:24:11.470294932Z [2025-05-04 14:24:11,467] [INFO] [kombu.mixins] Connected to redis://redis:6379/1
2025-05-04T14:27:35.109283837Z [2025-05-04 14:27:35,105] [ERROR] [django.request] Internal Server Error: /api/profile/
2025-05-04T14:27:35.109301304Z Traceback (most recent call last):
2025-05-04T14:27:35.109305249Z   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
2025-05-04T14:27:35.109308727Z     raise exc_info[1]
2025-05-04T14:27:35.109311880Z   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 42, in inner
2025-05-04T14:27:35.109315263Z     response = await get_response(request)
2025-05-04T14:27:35.109318390Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109321522Z   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
2025-05-04T14:27:35.109324743Z     raise exc_info[1]
2025-05-04T14:27:35.109327847Z   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
2025-05-04T14:27:35.109331150Z     response = await wrapped_callback(
2025-05-04T14:27:35.109334245Z                ^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109337346Z   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 468, in __call__
2025-05-04T14:27:35.109340495Z     ret = await asyncio.shield(exec_coro)
2025-05-04T14:27:35.109343563Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109350862Z   File "/usr/local/lib/python3.12/site-packages/asgiref/current_thread_executor.py", line 40, in run
2025-05-04T14:27:35.109354205Z     result = self.fn(*self.args, **self.kwargs)
2025-05-04T14:27:35.109357363Z              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109360479Z   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 522, in thread_handler
2025-05-04T14:27:35.109363683Z     return func(*args, **kwargs)
2025-05-04T14:27:35.109366755Z            ^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109369828Z   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
2025-05-04T14:27:35.109373079Z     return view_func(request, *args, **kwargs)
2025-05-04T14:27:35.109376165Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109379251Z   File "/usr/local/lib/python3.12/site-packages/django/views/generic/base.py", line 104, in view
2025-05-04T14:27:35.109387541Z     return self.dispatch(request, *args, **kwargs)
2025-05-04T14:27:35.109390319Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109392873Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
2025-05-04T14:27:35.109395502Z     response = self.handle_exception(exc)
2025-05-04T14:27:35.109397978Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109400539Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
2025-05-04T14:27:35.109403187Z     self.raise_uncaught_exception(exc)
2025-05-04T14:27:35.109406576Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
2025-05-04T14:27:35.109409299Z     raise exc
2025-05-04T14:27:35.109411830Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
2025-05-04T14:27:35.109414446Z     response = handler(request, *args, **kwargs)
2025-05-04T14:27:35.109418703Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109421283Z   File "/usr/src/paperless/src/paperless/views.py", line 184, in get
2025-05-04T14:27:35.109423891Z     return Response(serializer.to_representation(user))
2025-05-04T14:27:35.109426411Z                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109428959Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/serializers.py", line 538, in to_representation
2025-05-04T14:27:35.109431671Z     ret[field.field_name] = field.to_representation(attribute)
2025-05-04T14:27:35.109434235Z                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109436761Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/serializers.py", line 714, in to_representation
2025-05-04T14:27:35.109439411Z     self.child.to_representation(item) for item in iterable
2025-05-04T14:27:35.109441910Z     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109444390Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/serializers.py", line 538, in to_representation
2025-05-04T14:27:35.109447035Z     ret[field.field_name] = field.to_representation(attribute)
2025-05-04T14:27:35.109449540Z                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109452085Z   File "/usr/local/lib/python3.12/site-packages/rest_framework/fields.py", line 1870, in to_representation
2025-05-04T14:27:35.109454791Z     return method(value)
2025-05-04T14:27:35.109457288Z            ^^^^^^^^^^^^^
2025-05-04T14:27:35.109459824Z   File "/usr/src/paperless/src/paperless/serialisers.py", line 150, in get_name
2025-05-04T14:27:35.109462610Z     return obj.get_provider_account().to_str()
2025-05-04T14:27:35.109465153Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109469837Z   File "/usr/local/lib/python3.12/site-packages/allauth/socialaccount/models.py", line 153, in get_provider_account
2025-05-04T14:27:35.109472658Z     return self.get_provider().wrap_account(self)
2025-05-04T14:27:35.109475243Z            ^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109479429Z   File "/usr/local/lib/python3.12/site-packages/allauth/socialaccount/models.py", line 147, in get_provider
2025-05-04T14:27:35.109482072Z     provider = self._provider = adapter.get_provider(
2025-05-04T14:27:35.109484565Z                                 ^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109487069Z   File "/usr/local/lib/python3.12/site-packages/allauth/socialaccount/adapter.py", line 213, in get_provider
2025-05-04T14:27:35.109489774Z     app = self.get_app(request, provider=provider, client_id=client_id)
2025-05-04T14:27:35.109492617Z           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-05-04T14:27:35.109495260Z   File "/usr/local/lib/python3.12/site-packages/allauth/socialaccount/adapter.py", line 303, in get_app
2025-05-04T14:27:35.109497890Z     raise SocialApp.DoesNotExist()
2025-05-04T14:27:35.109500478Z allauth.socialaccount.models.SocialApp.DoesNotExist
2025-05-04T14:30:00.002952262Z [2025-05-04 14:30:00,002] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T14:30:00.010914860Z [2025-05-04 14:30:00,010] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[0dfab148-c41e-4237-ba62-e37a5cf89615] received
2025-05-04T14:30:00.286567002Z [2025-05-04 14:30:00,286] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[0dfab148-c41e-4237-ba62-e37a5cf89615] succeeded in 0.27081028699467424s: 'No new documents were added.'
2025-05-04T14:40:00.000693174Z [2025-05-04 14:40:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T14:40:00.003007839Z [2025-05-04 14:40:00,002] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[0b609e0b-1550-4e6f-91cd-64c944f40151] received
2025-05-04T14:40:00.238063668Z [2025-05-04 14:40:00,236] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[0b609e0b-1550-4e6f-91cd-64c944f40151] succeeded in 0.23189545499917585s: 'No new documents were added.'
2025-05-04T14:50:00.000501490Z [2025-05-04 14:50:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T14:50:00.002935632Z [2025-05-04 14:50:00,002] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[efb1f099-2277-4080-a589-ad8cfec0bac8] received
2025-05-04T14:50:00.280514064Z [2025-05-04 14:50:00,279] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[efb1f099-2277-4080-a589-ad8cfec0bac8] succeeded in 0.2752841030014679s: 'No new documents were added.'
2025-05-04T15:00:00.000556660Z [2025-05-04 15:00:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T15:00:00.004763789Z [2025-05-04 15:00:00,003] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[7bfd2b60-f10d-4591-a105-3190d082096f] received
2025-05-04T15:00:00.234040085Z [2025-05-04 15:00:00,233] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[7bfd2b60-f10d-4591-a105-3190d082096f] succeeded in 0.22756847200798802s: 'No new documents were added.'
2025-05-04T15:05:00.000701048Z [2025-05-04 15:05:00,000] [INFO] [celery.beat] Scheduler: Sending due task Train the classifier (documents.tasks.train_classifier)
2025-05-04T15:05:00.003409427Z [2025-05-04 15:05:00,003] [INFO] [celery.beat] Scheduler: Sending due task Check and run scheduled workflows (documents.tasks.check_scheduled_workflows)
2025-05-04T15:05:00.003620527Z [2025-05-04 15:05:00,003] [INFO] [celery.worker.strategy] Task documents.tasks.train_classifier[0eec3dae-0e75-4ac4-b22c-d11f034da1eb] received
2025-05-04T15:05:00.006382689Z [2025-05-04 15:05:00,005] [INFO] [celery.worker.strategy] Task documents.tasks.check_scheduled_workflows[c8ed7dc8-ba7e-4709-b6eb-2003cb05a717] received
2025-05-04T15:05:00.027231179Z [2025-05-04 15:05:00,026] [INFO] [celery.app.trace] Task documents.tasks.check_scheduled_workflows[c8ed7dc8-ba7e-4709-b6eb-2003cb05a717] succeeded in 0.017339368001557887s: None
2025-05-04T15:05:03.927723025Z [2025-05-04 15:05:03,926] [INFO] [paperless.classifier] No updates since last training
2025-05-04T15:05:03.963224435Z [2025-05-04 15:05:03,962] [INFO] [celery.app.trace] Task documents.tasks.train_classifier[0eec3dae-0e75-4ac4-b22c-d11f034da1eb] succeeded in 3.957507222003187s: None
2025-05-04T15:10:00.000672586Z [2025-05-04 15:10:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T15:10:00.003716057Z [2025-05-04 15:10:00,003] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[21888c30-ac64-462e-bee4-f2a616bd9771] received
2025-05-04T15:10:00.298712805Z [2025-05-04 15:10:00,295] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[21888c30-ac64-462e-bee4-f2a616bd9771] succeeded in 0.2894111170026008s: 'No new documents were added.'
2025-05-04T15:20:00.001453862Z [2025-05-04 15:20:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T15:20:00.003675489Z [2025-05-04 15:20:00,003] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[3ff1d8bd-d39e-4c0a-a20f-c6b9aeaad863] received
2025-05-04T15:20:00.345517503Z [2025-05-04 15:20:00,345] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[3ff1d8bd-d39e-4c0a-a20f-c6b9aeaad863] succeeded in 0.33682823900016956s: 'No new documents were added.'
2025-05-04T15:30:00.000552834Z [2025-05-04 15:30:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T15:30:00.003854330Z [2025-05-04 15:30:00,002] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[be211691-7733-419b-ad11-c4f1a3a71dad] received
2025-05-04T15:30:00.286305459Z [2025-05-04 15:30:00,285] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[be211691-7733-419b-ad11-c4f1a3a71dad] succeeded in 0.2806353279884206s: 'No new documents were added.'
2025-05-04T15:40:00.000565967Z [2025-05-04 15:40:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T15:40:00.003439813Z [2025-05-04 15:40:00,003] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[d89e9e11-ac50-4284-8e3f-d1c572660de0] received
2025-05-04T15:40:00.268043981Z [2025-05-04 15:40:00,267] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[d89e9e11-ac50-4284-8e3f-d1c572660de0] succeeded in 0.26194514500093646s: 'No new documents were added.'
2025-05-04T15:50:00.000597729Z [2025-05-04 15:50:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T15:50:00.003969105Z [2025-05-04 15:50:00,003] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[4891e45b-89ca-4ceb-bce1-7cfe62df97a1] received
2025-05-04T15:50:00.282626701Z [2025-05-04 15:50:00,282] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[4891e45b-89ca-4ceb-bce1-7cfe62df97a1] succeeded in 0.27647767700545955s: 'No new documents were added.'
2025-05-04T16:00:00.000791587Z [2025-05-04 16:00:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T16:00:00.003762557Z [2025-05-04 16:00:00,003] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[79d7f87b-f92a-453c-b73f-085204652a12] received
2025-05-04T16:00:00.311485824Z [2025-05-04 16:00:00,311] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[79d7f87b-f92a-453c-b73f-085204652a12] succeeded in 0.3051635250012623s: 'No new documents were added.'
2025-05-04T16:05:00.002047089Z [2025-05-04 16:05:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check and run scheduled workflows (documents.tasks.check_scheduled_workflows)
2025-05-04T16:05:00.011760263Z [2025-05-04 16:05:00,010] [INFO] [celery.beat] Scheduler: Sending due task Train the classifier (documents.tasks.train_classifier)
2025-05-04T16:05:00.012989816Z [2025-05-04 16:05:00,011] [INFO] [celery.worker.strategy] Task documents.tasks.check_scheduled_workflows[3fa76af8-7afa-459e-b7c2-349707089b6a] received
2025-05-04T16:05:00.025357364Z [2025-05-04 16:05:00,023] [INFO] [celery.worker.strategy] Task documents.tasks.train_classifier[e6e7ac10-efee-4d1d-bad4-35bfa54d4825] received
2025-05-04T16:05:00.053560673Z [2025-05-04 16:05:00,053] [INFO] [celery.app.trace] Task documents.tasks.check_scheduled_workflows[3fa76af8-7afa-459e-b7c2-349707089b6a] succeeded in 0.03144633899501059s: None
2025-05-04T16:05:03.121792743Z [2025-05-04 16:05:03,121] [INFO] [paperless.classifier] No updates since last training
2025-05-04T16:05:03.156307449Z [2025-05-04 16:05:03,155] [INFO] [celery.app.trace] Task documents.tasks.train_classifier[e6e7ac10-efee-4d1d-bad4-35bfa54d4825] succeeded in 3.119879495992791s: None
2025-05-04T16:10:00.001991591Z [2025-05-04 16:10:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T16:10:00.010067971Z [2025-05-04 16:10:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[6ae4a5bb-a107-4e67-8ff6-3c8fc8923478] received
2025-05-04T16:10:00.317610765Z [2025-05-04 16:10:00,317] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[6ae4a5bb-a107-4e67-8ff6-3c8fc8923478] succeeded in 0.3025092940079048s: 'No new documents were added.'
2025-05-04T16:20:00.001771576Z [2025-05-04 16:20:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T16:20:00.013083441Z [2025-05-04 16:20:00,011] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[388a2e3b-b6f0-4762-83ae-5b1dd71f4fca] received
2025-05-04T16:20:00.355953775Z [2025-05-04 16:20:00,355] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[388a2e3b-b6f0-4762-83ae-5b1dd71f4fca] succeeded in 0.3378118560067378s: 'No new documents were added.'
2025-05-04T16:30:00.000574764Z [2025-05-04 16:30:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T16:30:00.003358201Z [2025-05-04 16:30:00,002] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[b0940368-7444-4b55-b6df-6a2982ccd663] received
2025-05-04T16:30:00.281401445Z [2025-05-04 16:30:00,280] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[b0940368-7444-4b55-b6df-6a2982ccd663] succeeded in 0.2758072370052105s: 'No new documents were added.'
2025-05-04T16:40:00.001854512Z [2025-05-04 16:40:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T16:40:00.009821660Z [2025-05-04 16:40:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[26b90a59-9bb0-46d8-b413-ca8bd9cf36b3] received
2025-05-04T16:40:00.420109352Z [2025-05-04 16:40:00,419] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[26b90a59-9bb0-46d8-b413-ca8bd9cf36b3] succeeded in 0.4053170380066149s: 'No new documents were added.'
2025-05-04T16:50:00.001840009Z [2025-05-04 16:50:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T16:50:00.009626653Z [2025-05-04 16:50:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[4e46b517-f06b-4327-a244-9f0d082db79c] received
2025-05-04T16:50:00.323145346Z [2025-05-04 16:50:00,322] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[4e46b517-f06b-4327-a244-9f0d082db79c] succeeded in 0.307519570007571s: 'No new documents were added.'
2025-05-04T17:00:00.057877669Z [2025-05-04 17:00:00,057] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T17:00:00.061528423Z [2025-05-04 17:00:00,061] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[d4482d99-3655-4982-93bb-1f312f9ee2b9] received
2025-05-04T17:00:00.538944461Z [2025-05-04 17:00:00,538] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[d4482d99-3655-4982-93bb-1f312f9ee2b9] succeeded in 0.4751411489996826s: 'No new documents were added.'
2025-05-04T17:05:00.001691494Z [2025-05-04 17:05:00,000] [INFO] [celery.beat] Scheduler: Sending due task Train the classifier (documents.tasks.train_classifier)
2025-05-04T17:05:00.009640260Z [2025-05-04 17:05:00,008] [INFO] [celery.beat] Scheduler: Sending due task Check and run scheduled workflows (documents.tasks.check_scheduled_workflows)
2025-05-04T17:05:00.010109397Z [2025-05-04 17:05:00,008] [INFO] [celery.worker.strategy] Task documents.tasks.train_classifier[33877ea0-02ca-49d8-a4a7-4ec604fbc0f0] received
2025-05-04T17:05:00.017880725Z [2025-05-04 17:05:00,016] [INFO] [celery.worker.strategy] Task documents.tasks.check_scheduled_workflows[dd766caf-c16d-45d8-822c-c6d50da1af5e] received
2025-05-04T17:05:00.071844491Z [2025-05-04 17:05:00,070] [INFO] [celery.app.trace] Task documents.tasks.check_scheduled_workflows[dd766caf-c16d-45d8-822c-c6d50da1af5e] succeeded in 0.04781151299539488s: None
2025-05-04T17:05:03.131581394Z [2025-05-04 17:05:03,131] [INFO] [paperless.classifier] No updates since last training
2025-05-04T17:05:03.165900672Z [2025-05-04 17:05:03,165] [INFO] [celery.app.trace] Task documents.tasks.train_classifier[33877ea0-02ca-49d8-a4a7-4ec604fbc0f0] succeeded in 3.1496421030024067s: None
2025-05-04T17:10:00.001920004Z [2025-05-04 17:10:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T17:10:00.010127875Z [2025-05-04 17:10:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[d89159e9-5474-46d4-92ba-2977894e84c5] received
2025-05-04T17:10:00.365942185Z [2025-05-04 17:10:00,365] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[d89159e9-5474-46d4-92ba-2977894e84c5] succeeded in 0.35065469999972265s: 'No new documents were added.'
2025-05-04T17:20:00.001643780Z [2025-05-04 17:20:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T17:20:00.009812407Z [2025-05-04 17:20:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[641c0b60-7825-4c9e-abd6-9d1a8ccc21eb] received
2025-05-04T17:20:00.313386377Z [2025-05-04 17:20:00,312] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[641c0b60-7825-4c9e-abd6-9d1a8ccc21eb] succeeded in 0.2986106739990646s: 'No new documents were added.'
2025-05-04T17:30:00.001977435Z [2025-05-04 17:30:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T17:30:00.011989250Z [2025-05-04 17:30:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[da2a7335-b979-4e1c-b14c-2be6468236ac] received
2025-05-04T17:30:00.292143670Z [2025-05-04 17:30:00,291] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[da2a7335-b979-4e1c-b14c-2be6468236ac] succeeded in 0.279113630997017s: 'No new documents were added.'
2025-05-04T17:40:00.000998647Z [2025-05-04 17:40:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T17:40:00.005019918Z [2025-05-04 17:40:00,004] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[22f4f3da-6371-4b8c-9e72-d40410e0e5e9] received
2025-05-04T17:40:00.317569162Z [2025-05-04 17:40:00,316] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[22f4f3da-6371-4b8c-9e72-d40410e0e5e9] succeeded in 0.30891909000638407s: 'No new documents were added.'
2025-05-04T17:50:00.001755613Z [2025-05-04 17:50:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T17:50:00.009787552Z [2025-05-04 17:50:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[95bda74c-c8e9-41f5-8d03-2b7cd6fd4bca] received
2025-05-04T17:50:00.281279584Z [2025-05-04 17:50:00,279] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[95bda74c-c8e9-41f5-8d03-2b7cd6fd4bca] succeeded in 0.2645436839957256s: 'No new documents were added.'
2025-05-04T18:00:00.001822229Z [2025-05-04 18:00:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T18:00:00.009901795Z [2025-05-04 18:00:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[d823b87e-a3a7-446a-a23f-18dff86916a7] received
2025-05-04T18:00:00.319647762Z [2025-05-04 18:00:00,318] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[d823b87e-a3a7-446a-a23f-18dff86916a7] succeeded in 0.30447701399680227s: 'No new documents were added.'
2025-05-04T18:05:00.002091244Z [2025-05-04 18:05:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check and run scheduled workflows (documents.tasks.check_scheduled_workflows)
2025-05-04T18:05:00.009741183Z [2025-05-04 18:05:00,008] [INFO] [celery.beat] Scheduler: Sending due task Train the classifier (documents.tasks.train_classifier)
2025-05-04T18:05:00.010236314Z [2025-05-04 18:05:00,008] [INFO] [celery.worker.strategy] Task documents.tasks.check_scheduled_workflows[81430627-1103-4795-8eac-5aa63806b3b4] received
2025-05-04T18:05:00.017779453Z [2025-05-04 18:05:00,016] [INFO] [celery.worker.strategy] Task documents.tasks.train_classifier[f5b122a4-0c7b-4db2-9c04-830b6bc8831f] received
2025-05-04T18:05:00.055739149Z [2025-05-04 18:05:00,054] [INFO] [celery.app.trace] Task documents.tasks.check_scheduled_workflows[81430627-1103-4795-8eac-5aa63806b3b4] succeeded in 0.039256646996364s: None
2025-05-04T18:05:03.100271952Z [2025-05-04 18:05:03,099] [INFO] [paperless.classifier] No updates since last training
2025-05-04T18:05:03.134943128Z [2025-05-04 18:05:03,134] [INFO] [celery.app.trace] Task documents.tasks.train_classifier[f5b122a4-0c7b-4db2-9c04-830b6bc8831f] succeeded in 3.1123059520032257s: None
2025-05-04T18:10:00.001778961Z [2025-05-04 18:10:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T18:10:00.009925240Z [2025-05-04 18:10:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[e0a09d22-10d5-4574-8bb4-0374b6f01bc1] received
2025-05-04T18:10:00.313010788Z [2025-05-04 18:10:00,311] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[e0a09d22-10d5-4574-8bb4-0374b6f01bc1] succeeded in 0.2966531690035481s: 'No new documents were added.'
2025-05-04T18:20:00.002132332Z [2025-05-04 18:20:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T18:20:00.010304227Z [2025-05-04 18:20:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[e56a7988-2c98-492a-9b11-dd9e7109ba10] received
2025-05-04T18:20:00.317513349Z [2025-05-04 18:20:00,317] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[e56a7988-2c98-492a-9b11-dd9e7109ba10] succeeded in 0.30209617599030025s: 'No new documents were added.'
2025-05-04T18:30:00.001261269Z [2025-05-04 18:30:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T18:30:00.006729293Z [2025-05-04 18:30:00,005] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[5e518091-db14-4ccd-9a98-af7b25f8dd38] received
2025-05-04T18:30:00.401283670Z [2025-05-04 18:30:00,400] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[5e518091-db14-4ccd-9a98-af7b25f8dd38] succeeded in 0.39088164900022093s: 'No new documents were added.'
2025-05-04T18:40:00.001977888Z [2025-05-04 18:40:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T18:40:00.009913485Z [2025-05-04 18:40:00,008] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[ba4ea3ef-0fc9-45bd-a4bd-fbb92be88d2b] received
2025-05-04T18:40:00.411896347Z [2025-05-04 18:40:00,411] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[ba4ea3ef-0fc9-45bd-a4bd-fbb92be88d2b] succeeded in 0.3967932759987889s: 'No new documents were added.'
2025-05-04T18:45:00.001986800Z [2025-05-04 18:45:00,000] [INFO] [celery.beat] Scheduler: Sending due task Optimize the index (documents.tasks.index_optimize)
2025-05-04T18:45:00.009976130Z [2025-05-04 18:45:00,008] [INFO] [celery.worker.strategy] Task documents.tasks.index_optimize[444cd7c8-b350-46e2-8072-f6c1ad802397] received
2025-05-04T18:45:11.404288401Z [2025-05-04 18:45:11,403] [INFO] [celery.app.trace] Task documents.tasks.index_optimize[444cd7c8-b350-46e2-8072-f6c1ad802397] succeeded in 11.389016902001458s: None
2025-05-04T18:50:00.000526851Z [2025-05-04 18:50:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T18:50:00.002787583Z [2025-05-04 18:50:00,002] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[f1252e69-f97f-4142-bc68-d5244e2a2947] received
2025-05-04T18:50:00.286245748Z [2025-05-04 18:50:00,284] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[f1252e69-f97f-4142-bc68-d5244e2a2947] succeeded in 0.28071088099386543s: 'No new documents were added.'
2025-05-04T19:00:00.001715425Z [2025-05-04 19:00:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T19:00:00.008978315Z [2025-05-04 19:00:00,007] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[17ddb3cd-0aa7-4e1d-a56e-bcd9ef4ae847] received
2025-05-04T19:00:00.318729009Z [2025-05-04 19:00:00,317] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[17ddb3cd-0aa7-4e1d-a56e-bcd9ef4ae847] succeeded in 0.30489783200027887s: 'No new documents were added.'
2025-05-04T19:05:00.001838107Z [2025-05-04 19:05:00,000] [INFO] [celery.beat] Scheduler: Sending due task Train the classifier (documents.tasks.train_classifier)
2025-05-04T19:05:00.009440371Z [2025-05-04 19:05:00,008] [INFO] [celery.beat] Scheduler: Sending due task Check and run scheduled workflows (documents.tasks.check_scheduled_workflows)
2025-05-04T19:05:00.009914695Z [2025-05-04 19:05:00,008] [INFO] [celery.worker.strategy] Task documents.tasks.train_classifier[e24a3b85-2fc7-485e-994e-6fc7386c3975] received
2025-05-04T19:05:00.017281772Z [2025-05-04 19:05:00,016] [INFO] [celery.worker.strategy] Task documents.tasks.check_scheduled_workflows[c011db25-317b-4696-804f-3f7f1bbf8b2b] received
2025-05-04T19:05:00.052535473Z [2025-05-04 19:05:00,051] [INFO] [celery.app.trace] Task documents.tasks.check_scheduled_workflows[c011db25-317b-4696-804f-3f7f1bbf8b2b] succeeded in 0.030480925997835584s: None
2025-05-04T19:05:03.077797304Z [2025-05-04 19:05:03,077] [INFO] [paperless.classifier] No updates since last training
2025-05-04T19:05:03.128020256Z [2025-05-04 19:05:03,127] [INFO] [celery.app.trace] Task documents.tasks.train_classifier[e24a3b85-2fc7-485e-994e-6fc7386c3975] succeeded in 3.1131414489937015s: None
2025-05-04T19:10:00.000642047Z [2025-05-04 19:10:00,000] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
2025-05-04T19:10:00.003398866Z [2025-05-04 19:10:00,002] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[04829d99-737b-4ab1-b618-05ccd1157510] received
2025-05-04T19:10:00.312914640Z [2025-05-04 19:10:00,312] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[04829d99-737b-4ab1-b618-05ccd1157510] succeeded in 0.3073968090029666s: 'No new documents were added.'

--- a/src/documents/tests/test_api_profile.py
+++ b/src/documents/tests/test_api_profile.py
@@ -136,6 +136,36 @@ class TestApiProfile(DirectoriesMixin, APITestCase):
             ],
         )
 
+    def test_profile_w_social_removed_app(self):
+        """
+        GIVEN:
+            - Configured user and setup social account
+            - Social app has been removed
+        WHEN:
+            - API call is made to get profile
+        THEN:
+            - Profile is returned with "Unknown App" as name
+        """
+        self.setupSocialAccount()
+
+        # Remove the social app
+        SocialApp.objects.get(provider_id="keycloak-test").delete()
+
+        response = self.client.get(self.ENDPOINT)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        self.assertEqual(
+            response.data["social_accounts"],
+            [
+                {
+                    "id": 1,
+                    "provider": "keycloak-test",
+                    "name": "Unknown App",
+                },
+            ],
+        )
+
     def test_update_profile(self):
         """
         GIVEN:

--- a/src/paperless/serialisers.py
+++ b/src/paperless/serialisers.py
@@ -4,6 +4,7 @@ from allauth.mfa.adapter import get_adapter as get_mfa_adapter
 from allauth.mfa.models import Authenticator
 from allauth.mfa.totp.internal.auth import TOTP
 from allauth.socialaccount.models import SocialAccount
+from allauth.socialaccount.models import SocialApp
 from django.contrib.auth.models import Group
 from django.contrib.auth.models import Permission
 from django.contrib.auth.models import User
@@ -146,8 +147,11 @@ class SocialAccountSerializer(serializers.ModelSerializer):
             "name",
         )
 
-    def get_name(self, obj) -> str:
-        return obj.get_provider_account().to_str()
+    def get_name(self, obj: SocialAccount) -> str:
+        try:
+            return obj.get_provider_account().to_str()
+        except SocialApp.DoesNotExist:
+            return "Unknown App"
 
 
 class ProfileSerializer(serializers.ModelSerializer):

